tests/istio_testcases/example_policies/bookinfo-demo/bookinfo-test-request-attrs-scheme.yaml

```diff
@@ -41,6 +45,13 @@ queries:
       fwRulesGroupByLabelSinglePod: true
     expected: 0
 
+  - name: connectivity-istio-test-methods-basic-3
+    connectivityMap:
```

Comment:
I recommend testing with some other query types as well, with some comparison of a numeric result value (e.g. equiv or non-equiv, etc..). For the connectivityMap query there is no comparison of result values, and the output is not compared unless associated to output tests (in fw-rules tests). So this query here is more useful for some manual review of the actual output, and not for automatic testing to identify issues.

Reply:
Done